modules/reactor/src/meshgenerators/PinMeshGenerator.C

```diff
@@ -544,14 +637,17 @@ PinMeshGenerator::generate()
 
   // Update metadata at this point since values for these metadata only get set by PCCMG
   // at generate() stage
-  const auto max_radius_meta = getMeshProperty<Real>("max_radius_meta", name() + "_2D");
-  setMeshProperty("max_radius_meta", max_radius_meta);
-  const auto background_intervals_meta =
-      getMeshProperty<unsigned int>("background_intervals_meta", name() + "_2D");
-  setMeshProperty("background_intervals_meta", background_intervals_meta);
-  const auto node_id_background_meta =
-      getMeshProperty<dof_id_type>("node_id_background_meta", name() + "_2D");
-  setMeshProperty("node_id_background_meta", node_id_background_meta);
+  if (hasMeshProperty<Real>("max_radius_meta", name() + "_2D"))
+  {
```

Comment:
It d be a tad cleaner to check each one of them rather than rely on one for all the others

Reply:
Created separate if statements for each of these